src/main/charts/confluence/templates/statefulset-synchrony.yaml

```diff
@@ -49,6 +49,10 @@ spec:
             initialDelaySeconds: {{ .Values.synchrony.readinessProbe.initialDelaySeconds }}
             periodSeconds: {{ .Values.synchrony.readinessProbe.periodSeconds }}
             failureThreshold: {{ .Values.synchrony.readinessProbe.failureThreshold }}
+          {{- with .Values.synchrony.resources.container }}
```

Comment:
So the values for this already existed, but they were not put into the statefulset?

Reply:
Correct, they were just not utilised.